extensions/binding/org.eclipse.smarthome.binding.weatherunderground/ESH-INF/i18n/weatherunderground.properties

```diff
@@ -8,3 +8,6 @@ offline.conf-error-missing-location = The "location" parameter must be configure
 offline.conf-error-syntax-language = The value of the "language" parameter must contain 2 letters.
 offline.conf-error-min-refresh = The minimum value of the "refresh interval" parameter must be 5 minutes.
 offline.uri-error = Could not create URI to fetch data, please check if the location parameter is correct.
+
+# Discovery result
+discovery.weatherunderground.weather.local.label = Local Weather
```

Comment:
German translation is "Lokales Wetter"

Reply:
This was provided in an additional commit.